tests/test-cases/Group1-Docker-Commands/1-44-Docker-CP-Online.robot

```diff
@@ -126,27 +126,27 @@ Copy a directory from host to online container, dst is a volume
     Should Not Contain  ${output}  Error
     Should Contain  ${output}  bar
 
-Copy a file from host to offline container, dst is a volume shared with an online container
-    ${rc}  ${output}=  Run And Return Rc And Output  docker %{VCH-PARAMS} create -i --name offline -v vol1:/vol1 ${busybox}
+Copy a file from host to online container, dst is a volume shared with an online container
+    ${rc}  ${output}=  Run And Return Rc And Output  docker %{VCH-PARAMS} create -i --name online -v vol1:/vol1 ${busybox}
```

Comment:
The test tests for a copy to offline container with volume shared with an online container, that's why it's named offline. You can't name this container online because there's already an online container named online.